### PR TITLE
Portfolio contact form and finish mobile version.

### PR DESCRIPTION
**Pull request by maximoortelli**
_**Portfolio contact form and finish mobile version is an exercise where I will add a contact form to my portfolio with my mate Ahmed Alpanna. The goal here is to master all of the tools and best practices learned about in previous steps to use them in all Microverse projects and most likely in my future job as well.**_

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -655,7 +657,41 @@ <h4>LET'S CONNECT</h4>
...
-     </section>
+     </div>
+
+     <div class="contacto">
+
+         <div class="flex-column fondo">
+
+             <h2 class="margen">Contact me</h2>
+
+             <p class="margen">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
```
Comment (nahnahdev) [inline index.html]: - [x] Great job @maximoortelli and @ALPANNA-GH You have done a good job so far. There are just a few changes I will suggest you include and you'll be on your way to the next milestone. 
- [x] I noticed that there is a bigger spacing between the title `Contact me` and the paragraph text kindly close it up so it looks similar to the Figma Design :heavy_check_mark: 
- [x] Also, please ensure that there is similar spacing around these elements as well as seen in the Figma design :heavy_check_mark: 

| Your Design | The Figma Design |
|--------|--------|
| image | image |

File: index.html [hunk 2]
```diff
@@ -655,7 +657,41 @@ <h4>LET'S CONNECT</h4>
...
+                    
+                     <button id="boton" type="submit" >
+                        
+                         Get in touch
+
+                     </button>
+
+                 </div>
+
+             </form>
```
Comment (nahnahdev) [inline index.html:688]: - [x] Kindly ensure that your Form is properly centered on the page as seen in the Figma Design :+1: 

| Your Design | The Figma Design |
|--------|--------|
| image | image |

File: index.html [hunk 3]
```diff
@@ -655,7 +657,41 @@ <h4>LET'S CONNECT</h4>
...
+
+             <div id="ocult"><hr class="ocultarmob"></div>
+
+         </div>
+
```
Comment (nahnahdev) [inline index.html]: - [x] KIndly remove the white portion below your contact form :+1: 

| Your Design | The Figma Design |
|--------|--------|
| image | image |

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @maximoortelli and @ALPANNA-GH , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- Linters well setup and passing :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- The contact form works as expected :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (maximoortelli) [reply index.html]: Done, thank you NahnahAJ 👍😃

File: index.html (hunk 2 above)
Comment (maximoortelli) [reply index.html:688]: Done, thank you NahnahAJ 👍😃

File: index.html (hunk 3 above)
Comment (maximoortelli) [reply index.html]: Done, thank you NahnahAJ 👍😃

File: index.html (hunk 2 above)
Comment (maximoortelli) [reply index.html:688]: Done, thank you NahnahAJ 👍😃

**Review by DuaneDave — APPROVED** [review]
Hi @maximoortelli 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Your spacing is consistent with Figma
✔️ The styling is good

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**